Dockerfile

```diff
@@ -4,39 +4,27 @@ FROM node:18-buster AS base
 # Setup env variable for yarn
 ENV YARN_VERSION=4.3.1
 
-# Update dependencies, add python to the base image
-RUN apt-get update && apt-get install python3=3.7.3-1
-
-# Install and use yarn 4.x
-RUN corepack enable && corepack prepare yarn@${YARN_VERSION}
+# Update dependencies, add python to the base image, install and use yarn 4.x
+RUN apt-get update && apt-get install python3=3.7.3-1 && \
```

Comment:
For readability I'd do a `\` after each `&&`.

```
RUN apt-get update && \
   apt-get install python3=3.7.3-1 && \
   corepack enable && \
   corepack prepare yarn@${YARN_VERSION}
```

Reply:
done